docs/locale/fr/LC_MESSAGES/userguide/installation.po

```diff
@@ -0,0 +1,224 @@
+msgid ""
+"Windows installer is available at the latest stable version on `parsec.cloud "
+"<https://parsec.cloud/get-parsec>`_. Otherwise, it is possible to download a "
+"specific Parsec version on `GitHub <https://github.com/Scille/parsec/"
+"releases/latest>`_, for example to make Parsec work on a 32 bits computer "
+"(installers named ``parsec-vX.Y.Z-win32-setup.exe``)."
+msgstr ""
+"L'installeur Windows en dernière version stable est téléchargeable "
+"directement sur le site `parsec.cloud <https://parsec.cloud/get-parsec>`_. "
+"Autrement, il est possible de télécharger une version spécifique sur `GitHub "
+"<https://github.com/Scille/parsec/releases/latest>`_, par exemple pour faire "
+"fonctionner Parsec sur un ordinateur 32 bits (installeurs nommées ``parsec-"
+"vX.Y.Z-win32-setup.exe``)."
+
+#: ../../userguide/installation.rst:16
+msgid "Linux"
+msgstr ""
+
+#: ../../userguide/installation.rst:18
+msgid "Parsec is available on Snap:"
+msgstr "Parsec est disponible sur Snap:"
+
+#: ../../userguide/installation.rst:24
+msgid ""
+"If you are familiar with Snap, you may notice that Parsec snap is provided "
+"in classic mode (i.e. without sandbox). This is needed because Parsec needs "
+"`Fuse <https://en.wikipedia.org/wiki/Filesystem_in_Userspace>`_ to mount "
+"your data as a virtual directory, which is not allowed by the Snap sandbox."
+msgstr ""
+"Si vous connaissez Snap, vous avez peut-être remarqué que le snap Parsec "
+"fonctionne en mode classique (c'est à dire sans isolation). Cela est "
+"nécessaire car Parsec utilise `Fuse <https://fr.wikipedia.org/wiki/"
+"Filesystem_in_Userspace>`_ pour exposer vos données comme un disque virtuel, "
+"ce qui n'est pas autorisé par l'isolation de Snap."
+
+#: ../../userguide/installation.rst:29
+msgid "You can install the snap from the command line by doing:"
+msgstr "Vous pouvez installer le snap depuis la ligne de commande en faisant :"
+
+#: ../../userguide/installation.rst:37
+msgid ""
+"If you're using something other than ubuntu, you may need to install the "
+"package ``libfuse2`` or ``fuse2`` provided by your package manager (it may "
+"be already installed)."
+msgstr ""
+"Si vous utilisez autre chose que Ubuntu, vous devrez peut-être installé le "
+"paquet ``libfuse2`` ou ``fuse2`` fournit par votre gestionnaire de paquets "
+"(il est peut être déjà installé)."
+
+#: ../../userguide/installation.rst:40
+msgid ""
+"An indication that you're missing this package is if you can't mount a "
+"workspace to your filesystem."
+msgstr ""
+"Une indication que le paquet est manquant est si vous n'arrivez pas a monter "
+"un espace de travail dans votre systèmes de fichiers"
+
+#: ../../userguide/installation.rst:42
+msgid ""
+"In the log of the application, you will see something like ``fusermount: "
+"mount failed: Operation not permitted``."
+msgstr ""
+"Dans les logs de l'application, vous devrez voire quelque-chose comme "
+"``fusermount: mount failed: Operation not permitted``."
```

Comment:
Note: That text isn't present on `master`, I've added the translation is this PR.

Is it OK ? (look for typo, phrasing)